### ISSUE TITLE
Repository is not picked up from apps.config

### ISSUE THREAD

**Issue by jasoncorlett** [admpresales/nimbusapp#23]
With `apps.config` file:
```
nimbusapp-test.dockerapp jasoncorlett/nimbusapp-test:0.1.0
```

**Command:**
```
nimbusapp nimbusapp-test -d up
```

**Current Behaviour:**

Nimbusapp uses the default repository instead of the repository provided in the config file.

Output:
```
[demo@nimbusserver ~]$ nimbusapp nimbusapp-test -d up
Using nimbusapp-test.dockerapp:0.1.0 version found in /home/demo/.nimbusapp/apps.config
Running: docker-app render  admpresales/nimbusapp-test.dockerapp:0.1.0 > /home/demo/.nimbusapp/cache/nimbusapp-test/admpresales/nimbusapp-test.dockerapp/0.1.0/nimbusapp-test.dockerapp.yml
Running: docker-compose -p nimbusapp-test -f /home/demo/.nimbusapp/cache/nimbusapp-test/admpresales/nimbusapp-test.dockerapp/0.1.0/nimbusapp-test.dockerapp.yml up -d
nimbusapp-test-web is up-to-date
```

**Desired Behaviour:**

Nimbusapp should parse the image name in apps.config, and use the repository from that configuration file.

Expected Output:
```
Using nimbusapp-test.dockerapp:0.1.0 version found in /home/demo/.nimbusapp/apps.config
Running: docker-app render  jasoncorlett/nimbusapp-test.dockerapp:0.1.0 > /home/demo/.nimbusapp/cache/nimbusapp-test/jasoncorlett/nimbusapp-test.dockerapp/0.1.0/nimbusapp-test.dockerapp.yml
Running: docker-compose -p nimbusapp-test -f /home/demo/.nimbusapp/cache/nimbusapp-test/jasoncorlett/nimbusapp-test.dockerapp/0.1.0/nimbusapp-test.dockerapp.yml up -d
nimbusapp-test-web is up-to-date
```

**Workaround:**

Always specify the desired repository on the command line.

**Comment by jasoncorlett** [comment]
Good candidate to be fixed along with #18

**Comment by jhrabi** [comment]
@jasoncorlett Should this be closed?

**Comment by jasoncorlett** [comment]
@jhrabi I might be the only one who has encountered this issue, I've just been leaving it here as a reminder for next time I touch that section of code

**Comment by jhrabi** [comment]
K, I'll leave it open for now.

**Comment by jasoncorlett** [comment]
Fixed in release 1.3.0
